File: Python/1013-OMaior.py
import math
line=input().split(" ")
a, b, c=line
maior = (int(a)+int(b)+abs(int(a)-int(b)))/2
result = (int(maior)+int(c)+abs(int(maior)-int(c)))/2
print('{:.0f} eh o maior'.format(result))

File: Java/JavaScript/1013-OMaior.js
var input = require("fs").readFileSync("stdin", "utf8");
var line = input.split(" ");
var a = parseInt(line.shift());
var b = parseInt(line.shift());
var c = parseInt(line.shift());
maior=(a+b+Math.abs(a-b))/2;
result=(maior+c+Math.abs(maior-c))/2
console.log(`${result.toFixed(0)} eh o maior`);